js/core/src/reflection.ts

```diff
@@ -227,6 +232,15 @@ export class ReflectionServer {
       response.json(this.options.configuredEnvs);
     });
 
+    server.post('/api/notify', async (request, response) => {
+      const { telemetryServerUrl } = request.body;
+      if (typeof telemetryServerUrl === 'string') {
+        setTelemetryServerUrl(telemetryServerUrl);
+        logger.info(`Connected to telemetry server on ${telemetryServerUrl}`);
```

Comment:
`logger.info` is forbidden inside the core framework

```suggestion
        logger.debug(`Connected to telemetry server on ${telemetryServerUrl}`);
```